src/test/java/org/generationcp/middleware/service/impl/study/BMSAPIStudyServiceMock.java

```diff
@@ -0,0 +1,113 @@
+package org.generationcp.middleware.service.impl.study;
+
+import org.generationcp.middleware.service.api.phenotype.PhenotypeSearchDTO;
+import org.generationcp.middleware.service.api.phenotype.PhenotypeSearchRequestDTO;
+import org.generationcp.middleware.service.api.study.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.annotation.Resource;
+import java.util.List;
+@Service
+@Transactional
+public class BMSAPIStudyServiceMock implements StudyService {
```

Comment:
As discussed, we don't need to create this mock implementation for testing null value in autowiring